src/Sylius/Bundle/ResourceBundle/Controller/ResourceController.php

```diff
@@ -562,7 +562,7 @@ protected function isGrantedOr403($permission)
         if ($permission) {
             $grant = sprintf('%s.%s.%s', $this->config->getBundlePrefix(), $this->config->getResourceName(), $permission);
 
-            if (!$this->get('sylius.authorization_checker')->isGranted($grant)) {
+            if (!$this->get('security.authorization_checker')->isGranted($grant)) {
```

Comment:
I think we should pass `$resource` here, if it's available, so that ACL voters work properly.